browser/mocha/replace.js

```diff
@@ -22,6 +22,10 @@ extendInterfaces('replace', function(context, teardown) {
         // implementation for later:
         var originalInstanceTemplate = Polymer.Base.instanceTemplate;
 
+        if (Polymer.Base.instanceTemplate.isSinonProxy) {
```

Comment:
This is what I think the stub should look like:

```javascript
var replacements = {};

sinon.stub(Polymer.Base, 'instanceTemplate', function(template) {
  var dom = originalInstanceTemplate.apply(this, arguments);
  var nodeIterator = document.createNodeIterator(dom, NodeFilter.SHOW_ELEMENT);
  var node;

  while (node = nodeIterator.nextNode()) {
    var tagName = node.tagName.toLowercase();
    if (Object.hasOwnProperty(replacements, tagName)) {
      var replacement = document.createElement(replacements[tagName]);
      node.parentNode.replaceChild(replacement, node);
    }
  }
  return dom;
});
```

Besides stubbing `instanceTemplate` only on the first call, `with` would write into `replacements`.